test/runtime/specs/juttle-spec/modules/string/from-char-code.spec.md

```diff
@@ -0,0 +1,28 @@
+The `String.fromCharCode` function
+============================
+
+Produces an error when passed an argument of incorrect type
+-----------------------------------------------------------
+
+### Juttle
+
+    emit -from Date.new(0) -limit 1 | put result = String.fromCharCode(null) | view result
```

Comment:
I've seen this in a few other `juttle-spec` tests but is there a good reason why we tend to enumerate the negative tests and never add a positive test ? At least that way the coverage of that one line 27 above would happen unlike the current situation of this commit where that one line would be another "uncovered" line.